naturblick/utils/URL+Wikipedia.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright © 2024 Museum für Naturkunde Berlin.
+// This code is licensed under MIT license (see LICENSE.txt for details)
+
+import Foundation
+
+extension URL {
+    private static let deBaseUrl = "https://de.wikipedia.org"
+    private static let enBaseUrl = "https://en.wikipedia.org"
+
+        private static func wikiName(sciname: String) -> String? {
+            let tokens = sciname.split(separator: " ").prefix(2)
```

Comment:
Why `prefix(2)`?

Reply:
Sometimes scinames have the person responsible or other additional information after the name. This discards this data.

Reply:
ok, i just wondered if prefix(2) is always correct.